example/example.mocks.dart

```diff
@@ -0,0 +1,64 @@
+// Mocks generated by Mockito 5.0.6 from annotations
```

Comment:
Should I commit generated files?

Reply:
I have not been checking in the generated test file. You can go ahead and remove this file and I might reconsider later.